query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedConnectMutationInsideUpdateSpec.scala

```diff
@@ -7,72 +7,6 @@ import util._
 class NestedConnectMutationInsideUpdateSpec extends FlatSpec with Matchers with ApiSpecBase with SchemaBaseV11 {
   override def runOnlyForCapabilities: Set[ConnectorCapability] = Set(JoinRelationLinksCapability)
 
-  "A P1! to C1! relation with the child already in a relation" should "error when connecting since old required parent relation would be broken" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.